snap/bin/kong-wrapper.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh -e
+
```

Comment:
Is this file all new, or was the logic extracted from Kong's dockerfile and/or scripts in the Kong docker image?

Reply:
This file is all new.